app/[domain]/page.tsx

```diff
@@ -200,24 +210,78 @@ export default async function IndexPage({
+          <div className="max-w-lg text-sm">
             Once you have submitted your chosen handle above, it will be created
-            and you can now change it in the Bluesky app.<br/>
-            Go to Settings {">"} Advanced {">"} Change my handle.<br/> 
-            Select &quot;I have my own domain&quot; and enter{" "}
-            {newHandle ? `"${newHandle}"` : "your new handle"}.<br/>
-            Leave the setting on DNS Panel and ignore the text box as this is all set up automatically.<br/>
-            Finally, tap &quot;Verify DNS Record&quot;.
-          </p>
-          <p className="mt-6 max-w-lg text-sm">
-            This project was initially created by mozzius, and adapted by ChrisRid for the NAFO fellas.<br/>
-            If you like mozzius project, consider{" "}
-            <a href="https://github.com/sponsors/mozzius" className="underline">
-              sponsoring their work.
-            </a>
-            .
-          </p>
+            and you can now change it in the Bluesky app:
+            <br />
+            <div className="pl-4 pt-2">
+              <ol className="list-decimal">
+                <li>
+                  Go to Settings {">"} Advanced {">"} Change my handle.
+                </li>
+                <li>
+                  Select &quot;I have my own domain&quot; and enter{" "}
+                  {newHandle ? `"${newHandle}"` : "your new handle"}.<br />
+                </li>
+                <li>
+                  Leave the setting on DNS Panel and ignore the text box as this
+                  is all set up automatically.
+                </li>
+                <li>Finally, tap &quot;Verify DNS Record&quot;.</li>
+              </ol>
+            </div>
+          </div>
         </Stage>
+        <div className="max-w-lg text-sm">
+          <p className="mt-2 max-w-lg text-sm">
+            This Service is made for the NAFO fellas on Bluesky Visit their
+            official website and help support them.
+          </p>
+          <div className="py-4">
+            <Image src={ImageNAFO} width={512} height={400} alt="NAFO image" />
+          </div>
+          <div className="max-w-lg text-center">
+            <p className="text-sm">
+              To help support the hosting costs for this service:
+            </p>
+            <p className="py-2 text-center">
+              <a
+                href="https://buymeacoffee.com/chrisrid"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                buymeacoffee.com/chrisrid
+              </a>
+            </p>
+            <p>
+              (Please prioritize supporting Ukrainian charities / NAFO first)
+            </p>
+            <div className="py-4">
+              <div className="inline-block bg-white p-2">
+                <Image
+                  src={ImageCoffeQR}
+                  width={300}
+                  height={300}
+                  alt="CoffeQR image"
```

Comment:
Addition - Please could we make this image a link, so if a user clicks on it, it opens: https://buymeacoffee.com/chrisrid